mobile/src/main/java/rs/readahead/washington/mobile/views/fragment/forms/BlankFormsListFragment.kt

```diff
@@ -58,13 +58,7 @@ class BlankFormsListFragment :
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         initObservers()
-
-        if (!Preferences.isJavarosa3Upgraded()) {
-            model.showFab.postValue(false)
-            showJavarosa2UpgradeSheet()
-        } else {
-            listBlankForms()
-        }
```

Comment:
maybe just comment the code, not remove it ?

Reply:
Yeah, whatever, I've commented that part.